src/pages/Groups/Group2/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,132 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  font-size: 1.5rem;
+  border: 1px black;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`
+
+export const Screen = styled.div`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid red;
+  max-width: 400px;
+  min-width: 300px;
+  width: 50vw;
+  height: 100vh;
+`
+
+export const SearchBar = styled.div`
+  top: 0;
+  max-width: 400px;
+  min-width: 300px;
+  width: 50vw;
+  height: 70px;
+  border: 1px solid green;
+  border-radius: 0 0 15px 15px;
+  background-color: lightgreen;
+`
+
+export const SearchButton = styled.div`
+  position: fixed;
+  top: 0;
+  right: 38%; /*Not too sure about this*/
```

Comment:
if not sure u can test it? does removing it affect styling? if it does means its useful n u should keep it in. if not its not useful so can remove

Reply:
resolved with commit: https://github.com/rhdevs/rhdevs-frontend-training-AY2122/pull/19/commits/e0a5a69250cd4daec6d4d2881d5c3b33cb834372